tcl/pd_guiprefs.tcl

```diff
@@ -68,12 +68,13 @@ proc ::pd_guiprefs::init {} {
     switch -- $::platform {
         "Darwin" {
             set backend "plist"
-            # on macOS the domain should be the same as the bundle ID
-            catch {
-                set ::pd_guiprefs::domain [exec defaults read
-                                           [file join $::sys_guidir .. .. Info]
-                                           CFBundleIdentifier]
-            }
+            ## on macOS the domain should be the same as the bundle ID
```

Comment:
Should we bump the plugin version to 0.2 with this? It could;d *potentially* break things if a fork uses a different domain and relied on this "feature."

Reply:
No. 
This "feature" has never been part of a stable release. 

Also, it was meant as a way to ensure that the bundle id and the preferences domain are the same (by using a single source of truth). Now we are back to manually enforcing this.

Reply:
I agree. It's not really needed.

Reply:
just to clarify...

> Also, it was meant as a way to ensure that the bundle id and the preferences domain are the same (by using a single source of truth). Now we are back to manually enforcing this.

what i meant is: reading the bundleID and using it as the preferences domain, doesn't really change the current behaviour, where we just short-circuit the readout and replace it with a hardcoded value (which the same as the bundle ID)